aws-xray-recorder-sdk-core/src/test/java/com/amazonaws/xray/entities/EntityTest.java

```diff
@@ -0,0 +1,21 @@
+package com.amazonaws.xray.entities;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+import com.amazonaws.xray.AWSXRay;
+import java.time.Duration;
+import org.junit.jupiter.api.Test;
+
+class EntityTest {
+
+    @Test
+    void testTimeModuleSerialization() {
+        Segment seg = new SegmentImpl(AWSXRay.getGlobalRecorder(),"test");
+        Duration duration = Duration.ofSeconds(1);
```

Comment:
Can you do some test cases for smaller and larger values too? And also timestamps?

Reply:
Added!